data/json/effects.json

```diff
@@ -10,6 +10,14 @@
     "name": [ "Hit By Player" ],
     "desc": [ "AI tag for when monsters are hit by player.  This is a bug if you have it." ]
   },
+  {
+    "type": "effect_type",
+    "id": "took_damage",
+    "name": [ "Took Damage" ],
+    "desc": [
+      "AI tag for when a monster has taken damage.  Differs from the above by considering any possible source of damage. This is a bug if you have it."
```

Comment:
Player still has a chance to get this effect, and `Differs from the above` part would be totally unclear to him. I suggest rewriting this part to avoid any possible confusion.